addons/mrp/models/stock_move.py

```diff
@@ -242,6 +243,40 @@ def default_get(self, fields_list):
                 defaults['reference'] = production_id.name
         return defaults
 
+    @api.model_create_multi
+    def create(self, values):
```

Comment:
detail but maybe use vals_list since it's a multi. It's a bit disturbing since we often use values in create single and vals_list in create multi

Reply:
Whoops, fixed it.